### ISSUE TITLE
Starting job with "--use-job-endpoint" errors

### ISSUE THREAD

**Issue by styxit** [ansible/tower-cli#288]
I want to start a job with the recently added `--use-job-endpoint` option. However, just using this flag causes an error.

Seems like the problem is determining the `ignored_fields` in [tower_cli/resources/job.py#L221](https://github.com/ansible/tower-cli/blob/dbc4b746b8e316b16cf5146f299d5cc5928b4113/tower_cli/resources/job.py#L221).
```
ignored_fields = job_started.json().get('ignored_fields', {})
```
When i replace this line with `ignored_fields = {}` there are no errors.

I am using the latest tower-cli release `3.1.3` and my Tower version is `3.1.2`.

Example command:
```
$ tower-cli job launch --use-job-endpoint --job-template=5
```

Output:
```
$ tower-cli job launch --use-job-endpoint --job-template=5
Traceback (most recent call last):
  File "/usr/local/bin/tower-cli", line 90, in <module>
    cli()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/bin/tower-cli", line 82, in invoke
    return super(TowerCLI, self).invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/tower_cli/utils/decorators.py", line 41, in method_with_context_managed
    method(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/tower_cli/models/base.py", line 312, in func
    result = method(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/tower_cli/resources/job.py", line 221, in launch
    ignored_fields = job_started.json().get('ignored_fields', {})
  File "/usr/local/lib/python2.7/dist-packages/tower_cli/api.py", line 230, in json
    return super(APIResponse, self).json(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/models.py", line 826, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib/python2.7/json/__init__.py", line 351, in loads
    return cls(encoding=encoding, **kw).decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 384, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
```

**Comment by AlanCoding** [comment]
Fix will go out with 3.1.4. Thanks for finding this!

**Comment by styxit** [comment]
Thank you for the quick response and fix.
